python/lsst/ts/rubintv/handlers/ddv_websocket_handler.py

```diff
@@ -5,7 +5,9 @@
 import structlog
 from fastapi import APIRouter, WebSocket, WebSocketDisconnect
 
-ddv_ws_router = APIRouter()
+internal_ws_router = APIRouter()
+ddv_client_ws_router = APIRouter()
```

Comment:
Can this just be 1 `APIRouter`?

Reply:
This might related to a comment I'm leaving below.

Reply:
These definitely need to be separate as the internal one resolves without the app name in URL.